README.md

```diff
@@ -144,6 +144,20 @@ HLS features **not** supported:
 
 [mux.js]: https://github.com/videojs/mux.js/releases
 
+## MPEG-5 Part2 LCEVC Support
+
+**Only supported on browsers with Media Source Extensions SourceBuffer support**
```

Comment:
One question, do you want to support SmartTVs and Consoles in the future?

Reply:
Eventually yes.